css/style.css

```diff
@@ -1,4 +1,441 @@
+  top: 7px; 
+}
+
+.second-ln {
+  transform: translate(7px, -7px) rotate(-45deg); 
+  top: 21px;
+}
+
+.arrow:hover {
+  cursor: pointer;
+}
+
+
+@media (min-width:1200px) and (max-width:1295px) {
+  .menubar__link:nth-child(5) {
+    margin-right: 170px;
+  }
+  .tesla-bgd{
+    height: auto;
+    width: 100%;
+    max-width: 1295px;
+    top: -103px;
+    left: 0;
+  }
+  .tesla-car {
+    padding: 23px 0 20px 107px;
+  }
+  .content__data-car {
+    padding: 0 232px 22px 199px;
+  }
+  .data__speed-block {
+    padding-right: 55px;
+  }
+
+}
+@media (min-width:1024px) and (max-width:1199px) {
+  .menubar__link:nth-child(5) {
+    margin-right: 120px;
+  }
+  .tesla-bgd, .tesla-car {
+    height: auto;
+    width: 100%;
+    max-width: 1199px;
+  }
+  .tesla-bgd{
+    top: -75px;
+    left: 0;
+  }
+  .tesla-car {
+    padding: 65px 0 20px 4px;
+  }
+  .content__data-car {
+    padding: 0 calc(321px - 2 * (321px - (321 * 100% / 1440))) 22px;
+  }
+  .data__speed-block {
+    padding-right: 35px;
+  }
+  .test-btn {
+    padding-right: 0px;
+  }
+  .testdrive {
+    margin-right: 60px;
+  }
+  
+}
+@media (min-width:900px) and (max-width:1023px) {
+  .menubar__link {
+    display: none;
+  }
+  .tesla-bgd, .tesla-car {
+    height: auto;
+    width: 100%;
+    max-width: 1023px;
+  }
+  .tesla-bgd{
+    top: -35px;
+    left: 0;
+  }
+  .tesla-car {
+    padding: 54px 0 20px 4px;
+  }
+  .content__data-car {
+    padding: 0 calc(321px - 2 * (321px - (321 * 100% / 1440))) 22px;
+    flex-direction: column;
+  }
+  .content__facts {
+    padding-bottom: 20px;
+  }
+  .data__speed-icon {
+    width: 65px;
+    height: 50px;
+  }
+  .content__road-icon {
+    width: 43px;
+    height: 71px;
+  }
+  .data__figures {
+    font-size: 50px;
+  }
+  .data__specifications {
+    display: block;
+  }
+  .testdrive__img {
+    display: none;
+  }
+  .testdrive__text {
+    position: relative;
+    font-size: 16px;
+    color:  #e51937;
+  }
+  .testdrive__text:before {
+    position: absolute;
+    background-image: url("/img/png/steering-wheel-red.png");
+  }
+  .order-btn {
+    width: 221px;
+    height: 55px;
+    background-color: #e51937;
+    color: #fff;
+  }
+}
+@media (min-width:768px) and (max-width:899px) {
+  .header {
+    padding: 25px 24px 96px 0;
+  }
+  .header__logo-img{
+    padding-left: 15px;
+    width: 167px;
+    height: 50px;
+  }
+  .menubar__link {
+    display: none;
+  }
+  .button {
+    width: 33px;
+    height: 20px;
+  }
+  .second {
+    top: 10px;
+  }
+  .third {
+    top: 19px;
+  }
+  
+  .tesla-bgd, .tesla-car {
+    /* height: auto;
+    width: 100%;
+    max-width: 1023px; */
+  }
+  .tesla-bgd{
+    top: -22px;
+    width: 830px;
+    height: 271px;
+    padding-left: 35px;
+  }
+  .tesla-car {
+    width: 701px;
+    height: 332px;
+    padding: 85px 0 20px 33px;
+  }
+  .content__data-car {
+    flex-direction: column;
+  }
+}
+ @media (min-width:568) and (max-width: 768px) {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)